Source/Core/VideoBackends/Vulkan/VKTexture.cpp

```diff
@@ -21,6 +21,7 @@
 #include "VideoBackends/Vulkan/VKStreamBuffer.h"
 #include "VideoBackends/Vulkan/VulkanContext.h"
 
+#include <VideoCommon/DriverDetails.h>
 #include "VideoCommon/VideoConfig.h"
```

Comment:
Since this is dolphin header and not a library header, it should be quotes (i.e. `#include "VideoCommon/DriverDetails.h"`).